src/main/java/com/quantumsoft/qupathcloud/entities/Locations.java

```diff
@@ -18,15 +18,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The Locations class contains a list of {@link com.quantumsoft.qupathcloud.entities.Location}
+ * retrieving from the server.
```

Comment:
Locations contains a list of {@link com.quantumsoft.qupathcloud.entities.Location}
 retrieved from the server.